cli/installer/macos/installer/starfix-urlhandler.applescript

```diff
@@ -1,5 +1,5 @@
 on open location this_URL
-	set AppleScript's text item delimiters to ":"
+	set AppleScript's text item delimiters to "ide://"
```

Comment:
i don't think this change matters.

Reply:
Yes it doesn't . I misunderstood it earlier , although it worked . 
Later when I realized I converted it to a Draft PR (not ready for review) and thought that I'd keep pushing other changes  (if any) .

Reply:
so we can close this PR ?

Reply:
yes